### PR TITLE
Fix/json with bigints

### PR DESCRIPTION
**Pull request by sanjai0py**
# Description

https://github.com/usebruno/bruno/assets/84461672/a9b88d4b-ad16-4005-baaf-326934224e67

https://github.com/usebruno/bruno/assets/84461672/561de40d-937a-4ac3-9079-f5dd139b6de9

### Contribution Checklist:

- [x] **The pull request only addresses one issue or adds one feature.**
- [x] **The pull request does not introduce any breaking changes**
- [x] **I have added screenshots or gifs to help explain the change if applicable.**
- [x] **I have read the [contribution guidelines](https://github.com/usebruno/bruno/blob/main/contributing.md).**
- [x] **Create an issue and link to the pull request.**

Note: Keeping the PR small and focused helps make it easier to review and merge. If you have multiple changes you want to make, please consider submitting them as separate pull requests.

### Publishing to New Package Managers

Please see [here](../publishing.md) for more information.

### CODE REVIEW

File: packages/bruno-app/src/components/RequestPane/RequestBody/RequestBodyMode/index.js [hunk 1]
```diff
@@ -37,8 +38,8 @@ const RequestBodyMode = ({ item, collection }) => {
   const onPrettify = () => {
     if (body?.json && bodyMode === 'json') {
       try {
-        const bodyJson = JSON.parse(body.json);
-        const prettyBodyJson = JSON.stringify(bodyJson, null, 2);
+        const bodyJson = jsonBigint.parse(body.json);
```
Comment (Its-treason) [inline packages/bruno-app/src/components/RequestPane/RequestBody/RequestBodyMode/index.js:41]: This also needs to be done for the CLI package

File: packages/bruno-electron/src/ipc/network/prepare-request.js [hunk 2]
```diff
@@ -204,8 +205,7 @@ const prepareRequest = (request, collectionRoot, collectionPath) => {
       axiosRequest.headers['content-type'] = 'application/json';
     }
     try {
-      // axiosRequest.data = JSON.parse(request.body.json);
-      axiosRequest.data = JSON.parse(decomment(request.body.json));
+      axiosRequest.data = JSONbig.parse(decomment(request.body.json));
```
Comment (Its-treason) [inline packages/bruno-electron/src/ipc/network/prepare-request.js:208]: This also needs to be changed for the CLI

Comment (sanjai0py) [reply packages/bruno-electron/src/ipc/network/prepare-request.js:208]: Added for the CLI too. Thanks for the review @Its-treason

**Comment by helloanoop** [comment]
Thanks for taking care of this @sanjai0py !

**Pull request merged**